.github/workflows/icu_ci.yml

```diff
@@ -191,6 +191,27 @@ jobs:
           echo 'Running ICU4C intltest with stubdata.';
           INTLTEST_OPTS=-w make check 2>&1 | grep 'Elapsed Time: '
 
+  # Test U_CHARSET_IS_UTF8
+  u-charset-is-utf8-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          cd icu4c/source;
+          ./runConfigureICU Linux CPPFLAGS="-DU_CHARSET_IS_UTF8=1";
+          make -j2 check
+
+  # Test U_OVERRIDE_CXX_ALLOCATION-is-0-test
+  u-override-cxx-allocation-is-0-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          cd icu4c/source;
+          ./runConfigureICU Linux CPPFLAGS="-DU_OVERRIDE_CXX_ALLOCATION=0";
```

Comment:
Curiously the doc page for this test has the variable set before calling `./runConfigureICU`: 
http://site.icu-project.org/processes/release/tasks/healthy-code#TOC-Test-U_OVERRIDE_CXX_ALLOCATION-0

`CPPFLAGS="-DU_OVERRIDE_CXX_ALLOCATION=0" ./runConfigureICU Linux`
vs
`./runConfigureICU Linux CPPFLAGS="-DU_OVERRIDE_CXX_ALLOCATION=0";

Though perhaps it doesn't matter for this test.

Reply:
I assume that for environment variables, they have to come before the command, and that if you put them after, then the command sees them as garbage options. No? (FYI @srl295)

Reply:
> I assume that for environment variables, they have to come before the command, and that if you put them after, then the command sees them as garbage options. No? (FYI @srl295)

No, you can set variables afterwards this way.

It's a feature of `configure`.  Search for `*=*)` (as a literal, not a regex) in `configure`, or `configure --help`

> To assign environment variables ... specify them as VAR=VALUE.

Reply:
It does not matter. I ran runConfigureICU either way and the diffs were limited to the order of the flags in config.status in a few lines. The generated Makefiles have no differences and show the CPP flag at the compiler invokation.

I put the CPPFLAGS after the runConfigureICU to make it uniform with the U_CHARSET_IS_UTF8 test a few lines earlier.

Reply:
Thanks @srl295 and @gnrunge!